src/Foundatio/Extensions/DateTimeExtensions.cs

```diff
@@ -29,18 +39,18 @@ public static long ToUnixTimeSeconds(this DateTime date)
         return new DateTimeOffset(date.ToUniversalTime()).ToUnixTimeSeconds();
     }
 
-    public static DateTime FromUnixTimeSeconds(this long timestamp)
+    public static DateTimeOffset FromUnixTimeSeconds(this long timestamp)
     {
-        return DateTimeOffset.FromUnixTimeSeconds(timestamp).UtcDateTime;
+        return DateTimeOffset.FromUnixTimeSeconds(timestamp);
     }
 
-    public static DateTime SafeAdd(this DateTime date, TimeSpan value)
+    public static DateTimeOffset SafeAdd(this DateTimeOffset date, TimeSpan value)
     {
-        if (date.Ticks + value.Ticks < DateTime.MinValue.Ticks)
-            return DateTime.MinValue;
+        if (date.Ticks + value.Ticks < DateTimeOffset.MinValue.Ticks)
+            return DateTimeOffset.MinValue;
 
-        if (date.Ticks + value.Ticks > DateTime.MaxValue.Ticks)
-            return DateTime.MaxValue;
+        if (date.Ticks + value.Ticks > DateTimeOffset.MaxValue.Ticks)
+            return DateTimeOffset.MaxValue;
```

Comment:
probably a breaking change someplace we do not know. Should this all be internal?